homeassistant/components/html5/notify.py

```diff
@@ -164,19 +152,22 @@ def gcm_api_deprecated(value):
 )
 
 
-def get_service(
+async def async_get_service(
     hass: HomeAssistant,
     config: ConfigType,
     discovery_info: DiscoveryInfoType | None = None,
 ) -> HTML5NotificationService | None:
     """Get the HTML5 push notification service."""
+    if discovery_info is None:
+        return None
+
     json_path = hass.config.path(REGISTRATIONS_FILE)
 
     registrations = _load_config(json_path)
```

Comment:
This should do the job
```suggestion
    registrations = await hass.loop.run_in_executor(None, _load_config, json_path)
```

Reply:
Replaced with `await hass.async_add_executor_job`